### ISSUE TITLE
Trying to decrypt database files, getting this error

### ISSUE THREAD

**Issue by NatsukiOkusawa** [Francesco149/klbvfs#4]
As the title says, I'm trying to decrypt the database files. However, I keep getting this error:
![error](https://user-images.githubusercontent.com/71302813/93158143-889d4100-f746-11ea-88f3-70e681d36ad6.png)
Any idea what the problem is? Thank you.

**Comment by Francesco149** [comment]
try installing lxml, seems like an issue with your environment

**Comment by NatsukiOkusawa** [comment]
Thanks for the response. I just installed lxml but another error comes up:
![error](https://user-images.githubusercontent.com/71302813/93170866-fb1c1a00-f762-11ea-9037-db9fd5f66ea7.png)

**Comment by NatsukiOkusawa** [comment]
Okay, so looks like I managed to decrypt the database files. However, there's another problem:
![error](https://user-images.githubusercontent.com/71302813/93198862-8bb92100-f788-11ea-9509-436ca0fed83d.png)
Again, any idea what the problem is? Thank you.
If necessary, I will send you the decrypted database files I got.

**Comment by Francesco149** [comment]
I'm not sure. I haven't tested this since months ago so maybe they changed the encryption or something

**Comment by NatsukiOkusawa** [comment]
So, is there any chance for you to test it again, and possibly update your script if they did changed the encryption/whatever? If not, then I guess I'm out of luck then... :( Thank you for taking the time to help me though. :) And also, I almost forgot to say, but the decrypted database files I got would seem to open just fine in DB Browser for SQLite.

**Comment by NatsukiOkusawa** [comment]
Okay, nevermind. I finally got your script working. (The whole decryption/dump process took maaaaany hours though...) I got your script working by using the original, encrypted database files. (Turns out the script won't work if I use decrypted database files, I initially replaced the database files with the decrypted ones, thinking the script will work that way.) Once again, thank you very much for helping me. I highly appreciate your work/effort. Cheers! :D
